index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+     <!-- header -->
+     <div class="my-header">
+        <h1>Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+    </div>
+    <!-- image-section -->
+    <div class="image-section">
+        <div class="img"><img src="./images/1.jpg"/></div>
+        <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+    </div>
+
```

Comment:
It is a great practice to use semantic tags like header, nav over div tags. It would make your code more readable and SEO friendly if you wrap your h1 and p tag with header tag